tpot/base.py

```diff
@@ -1114,9 +1116,11 @@ def export(self, output_file_name, data_file_path=''):
                                     self.random_state,
                                     data_file_path=data_file_path)
 
-        with open(output_file_name, 'w') as output_file:
-            output_file.write(to_write)
-
+        if to_screen:
```

Comment:
Instead of printing the full text to stdout, maybe it is better to just return the string. User can easily print the returned string or grep information from it.